Project.toml

```
@@ -1,7 +1,7 @@
 name = "ConcurrentUtilities"
 uuid = "f0e56b4a-5159-44fe-b623-3e5288b988bb"
 authors = ["Jacob Quinn <quinn.jacobd@gmail.com>"]
-version = "2.4.3"
+version = "2.4.4"
```

this release is a new feature, not a bug fix:
```suggestion
version = "2.5.0"
```